tecken/settings.py

```diff
@@ -542,6 +542,15 @@ def dict_parser(val):
     default="/tmp/uploads",
     doc="The directory to use as a workspace for handling symbol uploads.",
 )
+UPLOAD_TEMPDIR_ORPHANS_CUTOFF = _config(
+    "UPLOAD_TEMPDIR_ORPHANS_CUTOFF",
+    default="60",
```

Comment:
It might be interesting to add parsers to Everett for handling things like "60m" or "5m30s" where it converts a time value to seconds. It could do the validation and parsing. It'd improve the readability of the configuration value. I'll write up an Everett feature for that.